src/test/test-json.c

```diff
@@ -751,4 +751,64 @@ TEST(json_array_append_nodup) {
         assert_se(json_variant_equal(s, nd));
 }
 
+TEST(bytes) {
+        _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
+        size_t n = 0;
+        uint8_t *p = NULL;
```

Comment:
Better to use `_cleanup_` attribute for `p` ? Though, this is testcode, hence not a big matter.